ts/parsers/graphicFrameParser.ts

```diff
@@ -0,0 +1,74 @@
+//Graphic frame node includes tables, charts and diagrams
+
+import { getAttributeByPath } from "../helpers/attributesHandler";
+
+export default class GraphicFrameParser {
+  public static processGraphicFrameNodes = (graphicFrames) => {
+    const result = [];
+    for (const frame of graphicFrames) {
+      const graphicTypeUri = getAttributeByPath(
+        [frame],
+        ["a:graphic", "a:graphicData", "$"]
+      ).uri;
+      switch (graphicTypeUri) {
+        case "http://schemas.openxmlformats.org/drawingml/2006/table":
+          result.push(frame);
+          break;
+        case "http://schemas.openxmlformats.org/drawingml/2006/chart":
+          break;
+        case "http://schemas.openxmlformats.org/drawingml/2006/diagram":
+          break;
+        default:
```

Comment:
empty default case? just double check to make sure you didn't need anything here

Reply:
we don't need anything, if there is something else we don't need anything, if its a table type we push the value to results cuz that's all we need.